mlir/lib/Dialect/SparseTensor/Transforms/SparseReinterpretMap.cpp

```diff
@@ -18,10 +20,135 @@ using namespace mlir::sparse_tensor;
 
 namespace {
 
-// TODO:
-//   (1) insert the zero-cost sparse_tensor.reinterpret_map ops
-//   (2) rewrite linalg.generic ops traits on level crds
-//   (3) compute topsort, and resolve cyles with sparse_tensor.convert ops
+//===----------------------------------------------------------------------===//
+// Helper methods.
+//===----------------------------------------------------------------------===//
+
+// Translates a "simple" map according to an identify lvl-map.
+static AffineMap translateMap(OpBuilder &builder, SparseTensorType stt,
```

Comment:
nit: I don't think you need `static` here as it is already inside an anonymous namespace

Reply:
Oh yeah, I am old-fashioned in the sense that "static" to me immediately flags the file-static property more so than the namespace context ;-) But as a result, I think it has become more or less style in this directory ;-)

Reply:
I triggered clang-tidy warning on my editor though. (but won't fail clang-format)